app/util.py

```diff
@@ -119,3 +126,27 @@ def project_user_permissions(form):
         permissions[user] = user_permissions
 
     return permissions
+
+
+# Padding for encryption
+# The lambdas are from StackOverflow, I have no idea how they work
+def pad(e):
```

Comment:
The standard flask way is to not encrypt anything ever, apparently. This is there so that the object to be encrypted will be the appropriate length (a multiple of 8, iirc), because otherwise the crypto library can't handle it. The library itself doesn't deal with that, and this was what I found to fix it.